BEAMS3D/Sources/beams3d_grid.f90

```diff
@@ -49,16 +61,12 @@ MODULE beams3d_grid
                      win_wall_load, win_wall_shine, win_hr, win_hp, win_hz, &
                      win_hri, win_hpi, win_hzi, win_NI5D, win_NI, &
                      win_raxis_fida, win_phiaxis_fida, win_zaxis_fida, win_energy_fida, win_pitch_fida, &
-                     win_beam_density, win_NEUTRONS, win_E_NEUTRONS, &
-                     nr_fida, nphi_fida, nz_fida, nenergy_fida, npitch_fida
-      REAL(rprec) :: rmin, rmax, zmin, zmax, phimin, phimax, tmin, tmax, delta_t, &
-                     vc_adapt_tol, psiedge_eq, phiedge_eq, plasma_Zmean, plasma_mass, &
-                     reff_eq, therm_factor, B_kick_min, B_kick_max, &
-      E_kick, freq_kick, t_fida, rho_fullorbit, rho_help,&
-      s_max,s_max_te, s_max_ne,s_max_zeff,s_max_ti, s_max_pot
+                     win_beam_density, win_NEUTRONS, win_E_NEUTRONS 
+      REAL :: emin_fida, pimin_fida
```

Comment:
Is it indented to not have `rprec` specified here explicitly?

Reply:
@kudav can you comment on this.

Reply:
Not intentional, I just pushed a fix.